packages/integration-tests/tests/api/accounts.test.ts

```diff
@@ -2,104 +2,262 @@ import * as request from "supertest";
+        expect(typeof response.body.result[0].input).toStrictEqual("string");
+        expect(typeof response.body.result[0].type).toStrictEqual("string");
+        expect(typeof response.body.result[0].contractAddress).toBeTruthy();
+        expect(typeof response.body.result[0].gasUsed).toStrictEqual("string");
+        expect(typeof response.body.result[0].fee).toStrictEqual("string");
+        expect(typeof response.body.result[0].l1BatchNumber).toStrictEqual("string");
+        expect(typeof response.body.result[0].traceId).toBeTruthy();
+        expect(typeof response.body.result[0].transactionType).toStrictEqual("string");
+        expect(typeof response.body.result[0].isError).toStrictEqual("string");
+        expect(typeof response.body.result[0].errCode).toStrictEqual("string");
+      });
+    });
+
+    //@id1804
+    it("Verify /api?module=account&action=txlistinternal", async () => {
+      await helper.retryTestAction(async () => {
+        const blocks = await request(environment.blockExplorerAPI).get("/blocks");
+        const blockNumber = blocks.body.items[0].number;
+        apiRoute = `/api?module=account&action=txlistinternal&page=1&offset=10&sort=desc&endblock=${blockNumber}&startblock=1`;
+        response = await helper.performGETrequest(apiRoute);
+
+        expect(response.status).toBe(200);
+        expect(response.body).toStrictEqual(expect.objectContaining({ status: "1" }));
+        expect(response.body).toStrictEqual(expect.objectContaining({ message: "OK" }));
+        expect(response.body.result.length).toBeGreaterThan(1);
+        expect(typeof response.body.result[0].blockNumber).toStrictEqual("string");
+        expect(typeof response.body.result[0].timeStamp).toStrictEqual("string");
+        expect(typeof response.body.result[0].hash).toBeTruthy(); //can be null
+        expect(typeof response.body.result[0].from).toStrictEqual("string");
+        expect(typeof response.body.result[0].to).toStrictEqual("string");
+        expect(typeof response.body.result[0].value).toStrictEqual("string");
+        expect(typeof response.body.result[0].input).toStrictEqual("string");
+        expect(typeof response.body.result[0].type).toStrictEqual("string");
+        expect(typeof response.body.result[0].traceId).toBeTruthy();
+        expect(typeof response.body.result[0].isError).toStrictEqual("string");
+        expect(typeof response.body.result[0].errCode).toStrictEqual("string");
+      });
+    });
+
+    //@id1805
+    it("Verify /api?module=account&action=tokentx", async () => {
+      await helper.retryTestAction(async () => {
+        const blocks = await request(environment.blockExplorerAPI).get("/blocks");
+        const blockNumber = blocks.body.items[0].number;
+        apiRoute = `/api?module=account&action=tokentx&page=1&offset=10&sort=desc&endblock${blockNumber}&startblock=0&contractaddress=${Token.ETHER_ERC20_Address}&address=${Wallets.richWalletAddress}`;
```

Comment:
do we need endblock=${blockNumber} instead of endblock${blockNumber} ?

Reply:
Fixed